.vscode/launch.json

```diff
@@ -62,6 +62,38 @@
         "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
       ]
     },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "l2 write client",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "program": "${workspaceFolder}/packages/client/src/write.ts",
+      "runtimeArgs": [
+        "-r",
+        "ts-node/register",
+        "-r",
+        "tsconfig-paths/register"
+      ],
+      "env": {
+        "NODE_ENV": "development",
+        "TS_NODE_PROJECT": "${workspaceFolder}/packages/client/tsconfig.json",
+      },
+      "console": "integratedTerminal",
+      "outFiles": [
+        "${workspaceFolder}/**/*.js",
+        "!**/node_modules/**"
+      ],
+      "args": [
+        "--resolver",
+        "0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512",
+        "-pk",
```

Comment:
```suggestion
        "--privateKey",
```

Reply:
there's a shorthand for this flag: https://github.com/blockful-io/external-resolver/pull/103/files#diff-fddecc9ce4eede7f9349805acc9070a43de231550d2b8c68a0c1ec32bac01e19R33